optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -564,7 +564,9 @@ private void writeWorstScoreCalculationSpeedDifferencePercentageSummaryChart() {
                 if (singleBenchmarkResult.hasAllSuccess()) {
                     double worstScoreCalculationSpeedDifferencePercentage
                             = singleBenchmarkResult.getWorstScoreCalculationSpeedDifferencePercentage();
-                    dataset.addValue(worstScoreCalculationSpeedDifferencePercentage, solverLabel, problemLabel);
+                    if (isFinite(worstScoreCalculationSpeedDifferencePercentage)) {
+                        dataset.addValue(worstScoreCalculationSpeedDifferencePercentage, solverLabel, problemLabel);
+                    }
                 }
```

Comment:
One more reason, I think, to extract that logic to a new method, and reuse that throughout the class.

Reply:
Yes, but I believe using lamdba's can help us extract the solverLabel creation, DefaultCategoryDataset line's etc too. Maybe not.
I still argue that is not blocking this PR - it's a separate issue.

@triceo Do you mind if I merge?